v1.0/Specification.md

```diff
@@ -184,11 +183,10 @@ The following table gives guidelines for how to comply with the requirements in
 | ODS_7 | Macros contain programming code that can execute and manipulate data in the spreadsheet and the file system. These may present security risks, and it is therefore prudent to remove any macros. Removal may result in loss of information, but typically macros are used to change data, and this goes against the intentions of preserving the data fixed in time, and removal may therefore often happen with no implications. |
-| ODS_8 | An OpenDocument file may be marked as “read only” in the file’s XML attribute “LoadReadonly” in “settings.xml”. This means the data won’t be editable unless a copy is created. |
-| ODS_9 | Printer settings are information related to local and/or network printers. This data may contain encrypted information on how to connect to the printer. For security reasons, this data may be removed. Removing the data will not result in the loss of any significant properties. |
-| ODS_10 | Metadata in the file property information and “meta.xml” such as author, title, category and user defined properties etc. can be misleading if i.e. the document was reused as a template. User defined custom properties may be created and stored in “meta.xml”.<br><br>You may consider removing this information from the file to not mislead any future user of the data.<br><br>Metadata can be saved in a sidecar file or in an associated table, to document their existence before removal. |
-| ODS_11 | Cell hyperlinks are in essence unproblematic to keep in a spreadsheet, but your organisation may want to remove cell hyperlinks, since they reference external information.<br><br>Cell hyperlinks may be opened using an internet archive, if the cell hyperlinks are no longer active.<br><br>Broken (unavailable URL) cell hyperlinks may be checked for upon ingest and reported.<br><br>Cell hyperlinks can be saved in a sidecar file or in an associated table, to document their existence before removal. |
-| ODS_12 | Preview image is a thumbnail image stored in the spreadsheet's zipped path “Thumbnails/thumbnail.png”. The preview image may be used by the operating system as the file's icon in the file browser/manager.<br><br>The preview image should be considered insignificant for preservation, since it may be generated at any given time and serves no essential purpose. It  may be considered an artefact. |
+| ODS_8 | An OpenDocument file may have a file called "settings.xml". This file stores XML attributes defining how an application should render the spreadsheet file. However, the "settings.xml" is not required by the ODF standard and it neither defines any XML attributes leaving the attributes entirely custom for the application, which created the file. Furthermore, only some applications creates or renders a "settings.xml" file.<br><br>The "settings.xml" file may be considered an artefact and either ignored or removed from the spreadsheet. |
```

Comment:
This seems to be a long requirement as it conflates the requirement and an edited version of its rationale. The language is a little woolly as a consequence. Requirements aren't the best place to try to shoehorn in contextual text.

Reply:
This is the guideline for the policy.